### ISSUE TITLE
New configuration proposal: hide buttons

### ISSUE THREAD

**Issue by alexisteleco** [Deejayfool/hass-shutter-card#87]
Hi, thanks for this amazing project!

## TL;DR:
My proposal is to add `none`/`hidden` value as an option to the `buttons_position` configuration, so you can do something like:

<details>
  <summary>Proposal configuration with three covers without buttons</summary>
  
```yaml
type: horizontal-stack
cards:
  - type: custom:shutter-card
    title: ''
    entities:
      - entity: cover.bedroom_blind
        name: Dormi
        buttons_position: hidden
        title_position: up
        shutter_width_px: 90
  - type: custom:shutter-card
    title: ''
    entities:
      - entity: cover.mari_blind
        name: Mari
        buttons_position: hidden
        title_position: up
        shutter_width_px: 90
  - type: custom:shutter-card
    title: ''
    entities:
      - entity: cover.alexis_blind
        name: Estudio
        buttons_position: hidden
        title_position: up
        shutter_width_px: 90
```
  
</details>

So you can get something like this:
<img width="381" alt="image" src="https://user-images.githubusercontent.com/17027081/226142067-e69656a4-f707-43e2-bdae-fee743e820d5.png">

## Full story:
I don't know if anyone will have the same problem as me, but my covers/shutter does not work with the close and open services, they only work by setting a specific number from 0 to 100.

Right now the picture works as expected, but the buttons does not work, and if you try to stop, open or close the blind by using them, an error message is presented to the user like this one:
```
Failed to call service cover/close_cover. Entity cover.bedroom_blind does not support this service.
```

There are people in my house that are not as technical as me, and they do not want to have to think were they can press or not, so _I'd like to completely remove the buttons_ and left only the picture with the nice slider.

Right now, I'm doing that using [card styles](https://github.com/thomasloven/lovelace-card-mod) (I leave my config below), but that is a rather convoluted way if you were not using card styles before

<details>
  <summary>Working configuration with three covers without buttons using [card styles](https://github.com/thomasloven/lovelace-card-mod)</summary>

```yaml
type: horizontal-stack
cards:
  - type: custom:shutter-card
    title: ''
    entities:
      - entity: cover.bedroom_blind
        name: Dormi
        buttons_position: right
        title_position: up
        shutter_width_px: 90
        disable_end_buttons: true
  - type: custom:shutter-card
    title: ''
    entities:
      - entity: cover.mari_blind
        name: Mari
        buttons_position: right
        title_position: up
        shutter_width_px: 90
        disable_end_buttons: true
  - type: custom:shutter-card
    title: ''
    entities:
      - entity: cover.alexis_blind
        name: Estudio
        buttons_position: right
        title_position: up
        shutter_width_px: 90
        disable_end_buttons: true
    style: |
      ha-card {
        font-size: 11px !IMPORTANT;
      }
      .sc-shutter-label {
        font-size: 15px !IMPORTANT;
      }
      .sc-shutter-buttons {
        display: none !IMPORTANT;
        visibility: hidden !IMPORTANT;
      }

```
</details>

**Comment by alexisteleco** [comment]
I just discovered that there is already a PR for this. I will mention this issue in the PR https://github.com/Deejayfool/hass-shutter-card/pull/42
